config.json

```diff
@@ -2173,6 +2253,42 @@
         "dependencyOnly": false,
         "templateSet": "tensorflow-lite"
       },
+      {
+        "groupId": "org.tensorflow",
+        "artifactId": "tensorflow-lite-task-audio",
+        "version": "0.4.3",
+        "nugetVersion": "0.4.3",
+        "nugetId": "Xamarin.TensorFlow.Lite.Task.Audio.Library",
```

Comment:
Is there a reason to add `.Library` to the `nugetId` of these 4 packages? Or is that a copy/paste error?

Reply:
Yes

https://www.nuget.org/packages?q=Xamarin.TensorFlow.Lite.Task

Another user published packages with Xamarin prefix. I tried to transfer ownership, but it was overly complicated.